src/applications/yellow-ribbon/constants/STATES.json

```diff
@@ -0,0 +1,206 @@
+[
+  {
+    "label": "Alabama",
+    "code": "AL"
+  },
+  {
+    "label": "Alaska",
+    "code": "AK"
+  },
+  {
+    "label": "Arizona",
+    "code": "AZ"
+  },
+  {
+    "label": "Arkansas",
+    "code": "AR"
+  },
+  {
+    "label": "California",
+    "code": "CA"
+  },
+  {
+    "label": "Colorado",
+    "code": "CO"
+  },
+  {
+    "label": "Connecticut",
+    "code": "CT"
+  },
+  {
+    "label": "Delaware",
+    "code": "DE"
+  },
+  {
+    "label": "District of Columbia",
+    "code": "DC"
+  },
+  {
+    "label": "Florida",
+    "code": "FL"
+  },
+  {
+    "label": "Georgia",
+    "code": "GA"
+  },
+  {
+    "label": "Hawaii",
+    "code": "HI"
+  },
+  {
+    "label": "Idaho",
+    "code": "ID"
+  },
+  {
+    "label": "Illinois",
+    "code": "IL"
+  },
+  {
+    "label": "Indiana",
+    "code": "IN"
+  },
+  {
+    "label": "Iowa",
+    "code": "IA"
+  },
+  {
+    "label": "Kansas",
+    "code": "KS"
+  },
+  {
+    "label": "Kentucky",
+    "code": "KY"
+  },
+  {
+    "label": "Louisiana",
+    "code": "LA"
+  },
+  {
+    "label": "Maine",
+    "code": "ME"
+  },
+  {
+    "label": "Maryland",
+    "code": "MD"
+  },
+  {
+    "label": "Massachusetts",
+    "code": "MA"
+  },
+  {
+    "label": "Michigan",
+    "code": "MI"
+  },
+  {
+    "label": "Minnesota",
+    "code": "MN"
+  },
+  {
+    "label": "Mississippi",
+    "code": "MS"
+  },
+  {
+    "label": "Missouri",
+    "code": "MO"
+  },
+  {
+    "label": "Montana",
+    "code": "MT"
+  },
+  {
+    "label": "Nebraska",
+    "code": "NE"
+  },
+  {
+    "label": "Nevada",
+    "code": "NV"
+  },
+  {
+    "label": "New Hampshire",
+    "code": "NH"
+  },
+  {
+    "label": "New Jersey",
+    "code": "NJ"
+  },
+  {
+    "label": "New Mexico",
+    "code": "NM"
+  },
+  {
+    "label": "New York",
+    "code": "NY"
+  },
+  {
+    "label": "North Carolina",
+    "code": "NC"
+  },
+  {
+    "label": "North Dakota",
+    "code": "ND"
+  },
+  {
+    "label": "Ohio",
+    "code": "OH"
+  },
+  {
+    "label": "Oklahoma",
+    "code": "OK"
+  },
+  {
+    "label": "Oregon",
+    "code": "OR"
+  },
+  {
+    "label": "Pennsylvania",
+    "code": "PA"
+  },
+  {
+    "label": "Rhode Island",
+    "code": "RI"
+  },
+  {
+    "label": "South Carolina",
+    "code": "SC"
+  },
+  {
+    "label": "South Dakota",
+    "code": "SD"
+  },
+  {
+    "label": "Tennessee",
+    "code": "TN"
+  },
+  {
+    "label": "Texas",
+    "code": "TX"
+  },
+  {
+    "label": "Utah",
+    "code": "UT"
+  },
+  {
+    "label": "Vermont",
+    "code": "VT"
+  },
+  {
+    "label": "Virginia",
+    "code": "VA"
+  },
+  {
+    "label": "Washington",
+    "code": "WA"
+  },
+  {
+    "label": "West Virginia",
+    "code": "WV"
+  },
+  {
+    "label": "Wisconsin",
+    "code": "WI"
+  },
+  {
+    "label": "Wyoming",
+    "code": "WY"
+  }
+]
```

Comment:
Hmm, should this app be using something that's in `profile360`? I feel like this would be useful to have in `platform/**` somewhere

Reply:
Yeah good question. There's also a bunch of `states` stuff in the API, such as here - https://department-of-veterans-affairs.github.io/va-digital-services-platform-docs/api-reference/#/benefits_info/getStates. Might be worth asking the other devs. There are likely differences across different products, but still, it would be cool to be sure